framework/src/media/audio/audio_manager.c

```diff
@@ -70,21 +70,16 @@
  * Private Types
  ****************************************************************************/
 enum audio_card_status_e {
-	AUDIO_CARD_NONE = 0,		// card not found yet, shouldn't be assigned explicitly.
+	AUDIO_CARD_NONE = 0, // card not found yet, shouldn't be assigned explicitly.
 	AUDIO_CARD_IDLE = 1,
 	AUDIO_CARD_READY,
 	AUDIO_CARD_RUNNING,
 	AUDIO_CARD_PAUSE
 };
 
-enum manufacturer_type_e {
-	AUDIO_CARD_REALTEK = 0
-};
+enum manufacturer_type_e { AUDIO_CARD_REALTEK = 0 };
```

Comment:
I already applied our cformatter.sh to audio_manager.c and audio_manager.h several times.
The existing cformatter is wrong...?